common/common/src/main/java/io/helidon/common/GenericType.java

```diff
@@ -66,22 +66,43 @@
     private final Class<?> rawType;
 
     /**
-     * Constructs a new generic type, using the provided generic type information and
-     * deriving the class.
+     * Constructs a new generic type, using the provided generic type
+     * information and deriving the class.
      *
      * @param genericType the generic type
-     * @param <N>         generic type of the returned GenericType
+     * @param <N> generic type of the returned GenericType
      * @return new type wrapping the provided type
```

Comment:
This seems to be just reformatted. Please do not reformat existing code unless necessary or an issue - e.g. the line 73 is OK, lines 69-70 not.